hack/run-e2e-kni.sh

```diff
@@ -10,7 +10,7 @@ PROJECT_DIR="${BASE_DIR}"/..
 
 RTE_NAMESPACE="${RTE_NAMESPACE:-rte-e2e}"
 E2E_NAMESPACE_NAME="${E2E_NAMESPACE_NAME:-rte-e2e}"
-E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-SingleNUMANodeContainerLevel}"
+E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-single-numa-node}"
```

Comment:
ok, CI passed because is not using the new scripts yet, right?

Reply:
Indeed.

Reply:
works for me.